todo_app/data/trello_items.py

```diff
@@ -20,18 +45,19 @@ def get_items():
     Returns: response json() containing trello card names / id's and lists id's
     """
 
-    url = 'https://api.trello.com/1/boards/' + boardid +'/cards?fields=name,idList,dateLastActivity'
+    url = 'https://api.trello.com/1/boards/' + str(get_boardid) +'/cards?fields=name,idList,dateLastActivity'
 
     query = {
-        'key': trello_key,
-        'token': trello_token
+        'key': str(get_trello_key),
+        'token': str(get_trello_token)
```

Comment:
Similarly here, and various other places below
```suggestion
        'key': str(get_trello_key()),
        'token': str(get_trello_token())
```